database/migrations/2022_04_04_152502_add_spotlight_to_osu_beatmapsets.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
update the date in filename

also the `osu_`/`Osu` prefix can be removed

(optionally change the class into anonymous class like in `2022_04_12_...` migration)